PurchasesCoreSwift/Logging/Strings/OfferingStrings.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  OfferingStrings.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/11/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCOfferingStrings) public class OfferingStrings: NSObject {
```

Comment:
not needed for this PR, but we should add a static instance of this class (and the others like it) to `Strings.swift`